### PR TITLE
Ac 264 add support duopro

### PR DESCRIPTION
**Pull request by roelandwyns**
More info about the specification : https://www.w3.org/Graphics/JPEG/itu-t81.pdf (page 35)
More info about how we started jumping segments:  (we made it a bit cleaner in our implementation imo) https://web.archive.org/web/20131016210645/http://www.64lines.com/jpeg-width-height

A really nice tool to work with these bytes: `Hex Fiend`

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -950,6 +963,54 @@ describe('parser (exif data)', () => {
...
+				assert.strictEqual(exif.GPSTimeStamp, '11:45:50')
+				assert.strictEqual(exif.GPSSpeedRef, 'K')
+				assert.strictEqual(exif.GPSSpeed, 0.52)
+				assert.strictEqual(exif.GPSTrackRef, 'T')
+				assert.strictEqual(exif.GPSTrack, 281.62)
+				assert.strictEqual(exif.GPSMapDatum, 'WGS84')
+				assert.strictEqual(exif.latitude, 51.389319444444446)
+				assert.strictEqual(exif.longitude, 12.511283333333333)
+
+                // XMP
```
Comment (rugheid) [inline test/test.js:1002]: I guess I messed up here? :P

**Review by rugheid — APPROVED** [review]
Looks good! 😛

File: test/test.js (hunk 1 above)
Comment (roelandwyns) [reply test/test.js:1002]: I will ident the comment once Oli took a look as well to the PR :)

File: test/test.js [hunk 2]
```diff
@@ -1800,10 +1873,15 @@ describe('parser (exif data)', () => {
...
 			assert.strictEqual(exif.xmp['crs:AlreadyApplied'], false)
+
+			// SOF
+			assert.strictEqual(exif.sof.ImageHeight, 3040)
+			assert.strictEqual(exif.sof.ImageWidth, 4056)
 		})
 
 		it('WingtraOne Sony DSC-RX1RII', async () => {
 			const exif = await parse(buffers['WingtraOne_Sony_DSC_RX1RII.JPG'], options)
+			console.log(exif);
```
Comment (OlivierKamers) [inline test/test.js]: Remove?

File: test/test.js [hunk 3]
```diff
@@ -578,7 +579,7 @@ describe('parser (exif data)', () => {
 
 	describe('cameras', () => {
 
-		const options = { xmp: true, flir: true }
+		const options = { xmp: true, flir: true, sof: true }
```
Comment (OlivierKamers) [inline test/test.js:582]: If the sof option is set to true but the exif data does not contain a sof segment, will it just be undefined?

**Review by OlivierKamers — APPROVED** [review]

File: test/test.js (hunk 3 above)
Comment (roelandwyns) [reply test/test.js:582]: Yes indeed :). 
image

File: test/test.js (hunk 1 above)
Comment (roelandwyns) [reply test/test.js:1002]: image
seems fine in webstorm :p

Comment (rugheid) [reply test/test.js:1002]: I think it's because we use spaces and they use tabs?

**Pull request merged**